### ISSUE TITLE
Aqara "open window sensor" MCCGQ11LM shows no temperature 

### ISSUE THREAD

**Issue by ardu1** [ioBroker/ioBroker.zigbee#984]
Hello,

I have 4 Aqara "open window sensors". I paired them with the Zigbee adapter under ioBroker. The ioBroker has the version 4.2.1, the Zigbee adapter has the version 1.4.3. and I use a Conbee II coordinator.

I do not found the temperature in the object list. 

Could anybody help?

Thanks and best regards !
![Screenshot-12-22-2021_07 22 43](https://user-images.githubusercontent.com/36503094/107737086-e2a92f00-6d03-11eb-8848-7f388971e8f2.jpg)

**Comment by markus2701** [comment]
The Sensors have NO Temperature Sensor on Board

**Comment by ardu1** [comment]
I think you are not right. Here is a picture from my friend, he uses the Deconz Adapter.

![image](https://user-images.githubusercontent.com/36503094/107780500-0c377a00-6d47-11eb-8456-3792e5ff217a.png)

**Comment by asgothian** [comment]
I can confirm what markus is writing. None of my sensors send temperature data.

If you are certain that your sensors do have a temperature measurement to report, please do the following:

- install my current master branch from [https://github.com/asgothian/ioBroker.zigbee](https://github.com/asgothian/ioBroker.zigbee)
- restart the adapter
- enter the IEEE of one of the sensors into the zigbee.0.info.debugmessages state
- watch the log for "warn" messages to appear when the sensor sends data. We can only generate a DP if the sensor actually sends temperature data.

I expect messages like these to show up in your log:
```
2021-02-14 11:44:43.030  - warn: zigbee.0 (7013) ELEVATED publishToState: message received '{"linkquality":75}' from device 00158d0002e20ffa type 'MCCGQ11LM'
2021-02-14 11:44:43.041  - warn: zigbee.0 (7013) ELEVATED publishToState: value generated '75' from device 00158d0002e20ffa for 'Link quality'
2021-02-14 11:44:43.048  - warn: zigbee.0 (7013) ELEVATED publishToState: message received '{"contact":false}' from device 00158d0002e20ffa type 'MCCGQ11LM'
2021-02-14 11:44:43.049  - warn: zigbee.0 (7013) ELEVATED publishToState: value generated 'false' from device 00158d0002e20ffa for 'Contact event'
2021-02-14 11:44:43.050  - warn: zigbee.0 (7013) ELEVATED publishToState: value generated 'true' from device 00158d0002e20ffa for 'Is open'
2021-02-14 11:44:46.583  - warn: zigbee.0 (7013) ELEVATED publishToState: message received '{"linkquality":81}' from device 00158d0002e20ffa type 'MCCGQ11LM'
2021-02-14 11:44:46.585  - warn: zigbee.0 (7013) ELEVATED publishToState: value generated '81' from device 00158d0002e20ffa for 'Link quality'
```

**Comment by ardu1** [comment]
![rmwjaajq](https://user-images.githubusercontent.com/36503094/107885672-f61eea80-6efb-11eb-9ad6-47c0e276306d.png)
This is an other Aqara window sensor, also using it with the Conbee II with the Deconz Adapter.

**Comment by ardu1** [comment]
@asgothian 
Could you please explain how to use your master branch exact?

**Comment by asgothian** [comment]
- go to the adapter page in iobroker admin
- switch to expert mode
- use the "install from GitHub" button
- select "custom"
- paste the link from above and confirm

**Comment by ardu1** [comment]
Could you read the necessary information.
host.ioBroker	2021-02-15 11:46:18.516	error	instance system.adapter.admin.0 terminated with code 1 (JS_CONTROLLER_STOPPED)
host.ioBroker	2021-02-15 11:46:18.516	error	Caught by controller[24]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.516	error	Caught by controller[24]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.516	error	Caught by controller[24]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.516	error	Caught by controller[24]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.516	error	Caught by controller[24]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[24]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[24]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[23]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[23]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[23]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[23]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[23]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[23]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[23]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[22]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.515	error	Caught by controller[22]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.514	error	Caught by controller[22]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.514	error	Caught by controller[22]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.514	error	Caught by controller[22]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.514	error	Caught by controller[22]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.514	error	Caught by controller[22]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.514	error	Caught by controller[21]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.514	error	Caught by controller[21]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.514	error	Caught by controller[21]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.514	error	Caught by controller[21]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.513	error	Caught by controller[21]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.513	error	Caught by controller[21]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.513	error	Caught by controller[21]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.513	error	Caught by controller[20]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.513	error	Caught by controller[20]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.513	error	Caught by controller[20]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.513	error	Caught by controller[20]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.513	error	Caught by controller[20]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.513	error	Caught by controller[20]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.513	error	Caught by controller[20]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.512	error	Caught by controller[19]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.512	error	Caught by controller[19]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.512	error	Caught by controller[19]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.512	error	Caught by controller[19]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.512	error	Caught by controller[19]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.512	error	Caught by controller[19]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.512	error	Caught by controller[19]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.512	error	Caught by controller[19]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.512	error	Caught by controller[19]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.511	error	Caught by controller[19]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.511	error	Caught by controller[19]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.511	error	Caught by controller[19]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.511	error	Caught by controller[19]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.511	error	Caught by controller[19]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.511	error	Caught by controller[18]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.511	error	Caught by controller[18]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.511	error	Caught by controller[18]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.511	error	Caught by controller[18]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.510	error	Caught by controller[18]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.508	error	Caught by controller[18]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.508	error	Caught by controller[18]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.508	error	Caught by controller[17]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.508	error	Caught by controller[17]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.508	error	Caught by controller[17]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.508	error	Caught by controller[17]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.508	error	Caught by controller[17]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.508	error	Caught by controller[17]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.508	error	Caught by controller[17]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.508	error	Caught by controller[16]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.507	error	Caught by controller[16]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.507	error	Caught by controller[16]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.507	error	Caught by controller[16]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.507	error	Caught by controller[16]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.507	error	Caught by controller[16]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.507	error	Caught by controller[16]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.507	error	Caught by controller[15]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.507	error	Caught by controller[15]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.507	error	Caught by controller[15]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.507	error	Caught by controller[15]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.506	error	Caught by controller[15]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.506	error	Caught by controller[15]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.506	error	Caught by controller[15]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.506	error	Caught by controller[14]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.506	error	Caught by controller[14]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.506	error	Caught by controller[14]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.506	error	Caught by controller[14]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.506	error	Caught by controller[14]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.506	error	Caught by controller[14]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.506	error	Caught by controller[14]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.505	error	Caught by controller[13]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.505	error	Caught by controller[13]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.505	error	Caught by controller[13]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.505	error	Caught by controller[13]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.505	error	Caught by controller[13]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.505	error	Caught by controller[13]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.505	error	Caught by controller[13]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.505	error	Caught by controller[12]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.505	error	Caught by controller[12]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.505	error	Caught by controller[12]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.504	error	Caught by controller[12]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.504	error	Caught by controller[12]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.504	error	Caught by controller[12]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.504	error	Caught by controller[12]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.504	error	Caught by controller[11]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.504	error	Caught by controller[11]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.504	error	Caught by controller[11]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.504	error	Caught by controller[11]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.504	error	Caught by controller[11]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.503	error	Caught by controller[11]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.503	error	Caught by controller[11]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.503	error	Caught by controller[10]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.503	error	Caught by controller[10]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.503	error	Caught by controller[10]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.503	error	Caught by controller[10]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.503	error	Caught by controller[10]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.503	error	Caught by controller[10]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.503	error	Caught by controller[10]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.503	error	Caught by controller[9]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.502	error	Caught by controller[9]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.502	error	Caught by controller[9]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.502	error	Caught by controller[9]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.502	error	Caught by controller[9]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.502	error	Caught by controller[9]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.502	error	Caught by controller[9]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.502	error	Caught by controller[8]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.502	error	Caught by controller[8]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.502	error	Caught by controller[8]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.501	error	Caught by controller[8]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.501	error	Caught by controller[8]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.501	error	Caught by controller[8]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.501	error	Caught by controller[8]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.501	error	Caught by controller[7]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.501	error	Caught by controller[7]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.501	error	Caught by controller[7]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.501	error	Caught by controller[7]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.501	error	Caught by controller[7]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.501	error	Caught by controller[7]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.500	error	Caught by controller[7]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.500	error	Caught by controller[6]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.500	error	Caught by controller[6]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.500	error	Caught by controller[6]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.500	error	Caught by controller[6]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.500	error	Caught by controller[6]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.500	error	Caught by controller[6]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.500	error	Caught by controller[6]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.499	error	Caught by controller[6]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.499	error	Caught by controller[6]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.499	error	Caught by controller[6]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.499	error	Caught by controller[6]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.499	error	Caught by controller[6]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.499	error	Caught by controller[6]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.499	error	Caught by controller[6]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.499	error	Caught by controller[5]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.499	error	Caught by controller[5]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.498	error	Caught by controller[5]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.498	error	Caught by controller[5]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.498	error	Caught by controller[5]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.498	error	Caught by controller[5]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.498	error	Caught by controller[5]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.498	error	Caught by controller[4]: at processTicksAndRejections (internal/process/task_queues.js:97:5)
host.ioBroker	2021-02-15 11:46:18.498	error	Caught by controller[4]: at runMicrotasks (<anonymous>)
host.ioBroker	2021-02-15 11:46:18.498	error	Caught by controller[4]: at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
host.ioBroker	2021-02-15 11:46:18.497	error	Caught by controller[4]: at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
host.ioBroker	2021-02-15 11:46:18.497	error	Caught by controller[4]: at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
host.ioBroker	2021-02-15 11:46:18.497	error	Caught by controller[4]: at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:6957:68
host.ioBroker	2021-02-15 11:46:18.497	error	Caught by controller[4]: TypeError: Cannot set property 'source' of undefined
host.ioBroker	2021-02-15 11:46:18.497	error	Caught by controller[3]: at processTicksAndRejections (internal/process/task_queues.js:97:5)

**Comment by Apollon77** [comment]
nutzt du alias? Da scheint einer komisch zu sein. starte mal alles neu. passierts dann immer noch? Ggf sonst mal js-controller 3.2 nutzen

**Comment by ardu1** [comment]
Ja, ich nutze Alias. An welcher Stelle sieht es komisch aus? Die Fenstersensoren laufen allerdings och nicht als Alias.

**Comment by ardu1** [comment]
Kannst Du mir verklickern, was der Zigbee Adapter gemacht hat? Sollte ich jetzt noch einmal den ursprünglichen laden?
Bin ins Terminal gegangen und habe nach updates gesucht, ist alles aktuell.

**Comment by ardu1** [comment]
Habe jetzt den ioBroker neu aus dem Backup gestartet und auch noch einmal alles auf updates überprüft. Jetzt laufen die Alias wieder richtig. Der js-controller hat die Version 3.1.6. und wird als aktuell angezeigt. Wie kann ich den updaten?

**Comment by asgothian** [comment]
Closed - apparently, the 1.4.4 adapter version is running. Please use the "ausschliessen" tab in the adapter configuration to obtain all states sent by the zigbee-herdsman-converters.

You will gain a temperature state, but i would be surprised if you will get any value.

**Comment by ardu1** [comment]
Habe jetzt die Option gefunden zum Updaten des js-controllers. Allerdings wird mir dort angezeigt, dass die 3.1.6 die aktuelle Version ist?
